src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -515,6 +537,8 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
     // cancel the previous debounce when inputValues changes to avoid race conditions
     // and always ensure the latest value is used
     return debounced.cancel
+    // We do not want to react on methods.getValues(), it's way too reacty and will cause THOR API spam
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
clean up old dependencies and add new `getWithdrawGasEstimateCryptoBaseUnit`. downside of the lint disable, wish you could pass it an arg for which hooks to disable
```
    accountId,
    asset,
    inputValues,
    opportunityData?.rewardsCryptoBaseUnit,
    opportunityData?.stakedAmountCryptoBaseUnit,
    getWithdrawGasEstimateCryptoBaseUnit,
    ```

Reply:
Very nice catch, yeah without the lint rule, it's not so easy to keep these in check! 😢 
eac6d2a20